core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/SignForIdentifierHandler.java

```diff
@@ -59,11 +67,21 @@ public void handle(final RoutingContext routingContext) {
       signerForIdentifier
           .sign(normaliseIdentifier(identifier), data)
           .ifPresentOrElse(
-              signature ->
+              signature -> {
+                if (slashingProtection.isPresent()) {
+                  if (slashingProtection.get().maySignAttestation(null, null, null)) {
+                    routingContext
+                        .response()
+                        .putHeader(CONTENT_TYPE, TEXT_PLAIN_UTF_8)
+                        .end(signature);
+                  }
```

Comment:
Is it a good idea to put an else here with a TODO?. This is the spot which should raise appropriate http error code (as per the spec).